ptbr/training_cli.py

```
@@ -0,0 +1,1406 @@
+    # Lazy-import heavy dependencies so validation stays fast
+    import json
+
+    import torch
+
+    from gliner import GLiNER
+    from ptbr.data import load_data
+
+    # -- Seed --
+    seed = cfg["run"]["seed"]
+    torch.manual_seed(seed)
+    run_name = cfg["run"].get("name", "")
```

Ruff (0.15.1)

[error] Local variable `run_name` is assigned to but never used

Remove assignment to unused variable `run_name`

(F841)